server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -95,3 +76,55 @@ describe('Support Needs tab view', () => {
     expect($('h2').text()).toEqual('Sorry, the Curious service is currently unavailable.')
   })
 })
+
+declare module 'cheerio' {
```

Comment:
This is how you extend cheerio - basically you declare additional methods on the interface ....